bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/async/AbstractRetryingOperation.java

```diff
@@ -263,6 +265,19 @@ protected boolean isRequestRetryable() {
     return rpc.isRetryable(getRetryRequest());
   }
 
+  private boolean isRstStream(Status status) {
+    if (!(this instanceof RetryingReadRowsOperation) || status.getCode() != Code.INTERNAL) {
```

Comment:
I think we can improve the isntanceof here

Maybe create a new method `isRetryable(Status)` which is:
```
protected boolean isRetryable(Status s)  {
  return retryOptions.isRetryable(code);
}
```
and override it in RetryReadRowsoperation:
@Override
protected boolean isRetryable(Status s)  {
  return  isRstStream(Status status) || super.isRetryable(code);
}
```